tests/v2/test_peer_manager.nim

```diff
@@ -24,7 +25,8 @@ import
   ../../waku/v2/protocol/waku_store,
   ../../waku/v2/protocol/waku_filter,
   ../../waku/v2/protocol/waku_swap/waku_swap,
-  ../test_helpers
+  ../test_helpers,
+  ./testlib/testutils
```

Comment:
If you don't have any `xtest` or `xsuite`, this will raise an "UnusedImport" warning.